src/components/Tooltip/Tooltip.stories.mdx

```diff
@@ -0,0 +1,225 @@
+import { withDesign } from "storybook-addon-designs";
+
+import Tooltip from "./Tooltip";
+import Button from "../Button/Button";
+import Radio from "../Radio/Radio";
+import RadioGroup from "../RadioGroup/RadioGroup";
+import Icon from "../Icons/Icon";
+import Link from "../Link/Link";
+import Label from "../Label/Label";
+import Image from "../Image/Image";
+import { VStack } from "@chakra-ui/react";
+import { getCategory } from "../../utils/componentCategories";
+import DSProvider from "../../theme/provider";
+
+<Meta
+  title={getCategory("Tooltip")}
+  component={Tooltip}
+  decorators={[withDesign]}
+  parameters={{
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=36835%3A25702",
+    },
+    jest: ["Tooltip.test.tsx"],
+  }}
+  argTypes={{
+    children: { control: false },
+    className: { control: false },
+    id: { control: false },
+    isDisabled: {
+      table: { defaultValue: { summary: false } },
+    },
+  }}
+/>
+
+# Tooltip
+
+| Component Version | DS Version |
+| ----------------- | ---------- |
+| Added             | `1.1.0`    |
+| Latest            | `1.1.0`    |
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Component Props](#component-props)
+- [Accessibility](#accessibility)
+- [Examples](#examples)
+
+## Overview
+
+<Description of={Tooltip} />
+
+The `Tooltip` component shows contextual help or information about a specific
+component when a user hovers or focuses on the component.
+
+## Component Props
+
+<Canvas withToolbar>
+  <Story
+    name="Tooltip with Controls"
+    args={{
+      children: undefined,
+      className: undefined,
+      content:
+        "This is a tooltip with information describing the hovered object.",
+      id: "tooltip-id",
+      isDisabled: false,
+    }}
+  >
+    {(args) => (
+      <Tooltip {...args}>
+        <Button id="tooltip-button">Hover me</Button>
+      </Tooltip>
+    )}
+  </Story>
+</Canvas>
+
+<ArgsTable story="Tooltip with Controls" />
+
+## Accessibility
+
+The DS `Tooltip` component is implemented with Chakra UI and Chakra handles the
+appropriate aria-describedby attribute for the element that the tooltip is associated with.
+
+The `Tooltip` component is accessible via keyboard. The color contrast between
+foreground color and background color is 4.5:1. If text size is 200%, the
+button scales with text so there is no overlap.
+
+If a tooltip is associated with a form control (text, radio, checkbox, etc),
+the text for the tooltip should be included in the form control label.
+
+Resources:
+
+- [W3C WAI ARIA Tooltip](https://www.w3.org/WAI/ARIA/apg/patterns/tooltip/)
+- [MDN ARIA: tooltip role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role)
+- [Chakra UI Tooltip](https://chakra-ui.com/docs/components/overlay/tooltip)
+
+## Examples
+
+### Tooltips on form options
+
+When using the `Tooltip` component in forms containing `Checkbox`, `Radio`, `Toggle`.
```

Comment:
I think this needs one or two sentences explaining that the `shouldWrapChildren` prop should be added for form components and _why_.